### PR TITLE
Use `gh issue edit` instead of `actions/github-script` to add labels in "Adding labels to issues"

### PR DESCRIPTION
**Pull request by jcbhmr**
### Why:

Closes: 

it's more clear to users -- using CLI tools instead of javascript (another language other than whatever the current project is using) with weird globals that are preassigned. it's much easier to quickly grasp whats going on with the gh cli than to try and track down first the docs for the actions/github-script and then the API docs for `octokit.rest.addLabel()` or whatever.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

changing from using the https://github.com/actions/github-script action to using the github cli (gh).

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by jcbhmr** [comment]
works great!
image

File: content/actions/managing-issues-and-pull-requests/adding-labels-to-issues.md [hunk 1]
```diff
@@ -44,20 +44,17 @@ In the tutorial, you will first make a workflow file that uses the [`actions/git
...
+              LABELS: triage
     ```
 
-1. Customize the `script` parameter in your workflow file:
-   - The `issue_number`, `owner`, and `repo` values are automatically set using the `context` object. You do not need to change these.
-   - Change the value for `labels` to the list of labels that you want to add to the issue. Separate multiple labels with commas. For example, `["help wanted", "good first issue"]`. For more information about labels, see "[AUTOTITLE](/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-labels-to-issues-and-pull-requests)."
+1. Customize the `env` values in your workflow file:
+   - The `GITHUB_TOKEN`, `GH_REPO`, and `NUMBER` values are automatically set using the `github` and `secrets` contexts. You do not need to change these.
+   - Change the value for `LABELS` to the list of labels that you want to add to the issue. Separate multiple labels with commas. For example, `help wanted,good first issue`. For more information about labels, see "[AUTOTITLE](/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-labels-to-issues-and-pull-requests)."
```
Comment (hubwriter) [inline content/actions/managing-issues-and-pull-requests/adding-labels-to-issues.md:57]:
```suggestion
   - Change the value for `LABELS` to the list of labels that you want to add to the issue. The label(s) must exist for your repository. Separate multiple labels with commas. For example, `help wanted,good first issue`. For more information about labels, see "[AUTOTITLE](/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-labels-to-issues-and-pull-requests)."
```

**Review by hubwriter — APPROVED** [review]
I really like this change.

Nice work @jcbhmr! 👏 

I also tested this and it works just fine. I changed one thing: I added a sentence stating that the label(s) the workflow adds must exist for this repo.

I'll get this merged/published. Many thanks for helping improve the docs. Much appreciated.

🚀

**Comment by waheedullah5961** [comment]
Thank you for supporting 👍

**Pull request merged**